server/googleActions.js

```diff
@@ -230,6 +234,7 @@ GoogleActions = {
 			updatedAt : new Date(),
 			gDrive    : true,
 			googleId  : obj.data.id,
+			pageCount : fileMetadata.properties.pageCount || 1,
```

Comment:
```suggestion
			pageCount : fileMetadata.properties.pageCount,
```
Same comments as above

Reply:
Committed.